### PR TITLE
add tag prefix to versioneer setup

### PR DESCRIPTION
**Pull request by rwblair**

### CODE REVIEW

File: bids-validator/bids_validator/__init__.py [hunk 1]
```diff
@@ -4,3 +4,7 @@
 __version__ = get_versions()['version']
 __all__ = ['BIDSValidator']
 del get_versions
+
+from ._version import get_versions
+__version__ = get_versions()['version']
+del get_versions
```
Comment (effigies) [inline bids-validator/bids_validator/__init__.py]: Doubled up. Undo this change.

File: bids-validator/bids_validator/__init__.py [hunk 2]
```diff
@@ -4,3 +4,7 @@
 __version__ = get_versions()['version']
 __all__ = ['BIDSValidator']
```
Comment (effigies) [inline bids-validator/bids_validator/__init__.py]: Can you manually verify that `bids_validator.__version__` works? If not, do:

```suggestion
__all__ = ['BIDSValidator', '__version__']
```

Comment (rwblair) [reply bids-validator/bids_validator/__init__.py]: worked for me

**Comment by effigies** [comment]
Cool. Thanks for this. Hopefully the next tag will update the website.
